test/unit/spec/common/jobManager-Spec.js

```diff
@@ -2152,5 +2079,181 @@ define([
                 expect(Jobs.getFsmStateFromJobs.calls.allArgs()).toEqual([[[]]]);
             });
         });
+
+        describe('requestBatchUpdate', () => {
+            beforeEach(function () {
+                this.jobManagerInstance = createJobManagerInstance(
+                    this,
+                    BatchMixin(JobManagerCore)
+                );
+            });
+
+            it('requests the status of active jobs', function () {
+                spyOn(this.jobManagerInstance.bus, 'emit');
+                this.jobManagerInstance.requestBatchStatus();
+                expect(this.bus.emit.calls.allArgs()).toEqual([
+                    [
+                        jcm.MESSAGE_TYPE.STATUS,
+                        { [jcm.PARAM.JOB_ID_LIST]: jasmine.arrayWithExactContents(ACTIVE_JOB_IDS) },
+                    ],
+                ]);
+            });
+
+            it('does not make a request if there are no active jobs', function () {
+                const terminalJobs = JobsData.allJobs.filter((job) => {
+                    return !ACTIVE_JOB_IDS.includes(job.job_id);
+                });
+                terminalJobs.push(JobsData.batchParentJob);
+                Jobs.populateModelFromJobArray(this.jobManagerInstance.model, terminalJobs);
+                spyOn(this.jobManagerInstance.bus, 'emit');
+                this.jobManagerInstance.requestBatchStatus();
+                expect(this.bus.emit.calls.allArgs()).toEqual([]);
```

Comment:
no job status request if the batch parent job is the only non-terminal job in the batch